### DISCUSSION TITLE
Build for older Macs

### DISCUSSION THREAD

**Discussion by Sh-nassau** [Alex313031/thorium#320] [General]
Is it possible to build an install for pre-Catharina MacOS?

**Comment by gz83** [comment]
@midzer @Alex313031

**Comment by midzer** [comment]
```
To use Chrome browser on Mac, you'll need:

macOS Catalina 10.15 or later
```
via https://support.google.com/chrome/a/answer/7100626

So I doubt it's possible with current versions.

**Comment by Alex313031** [comment]
@Sh-nassau You can use https://github.com/blueboxd/chromium-legacy

**Reply by programmingkidx** [reply]
I was impressed to find out that Thorium can run on an operating system as old as Windows XP. It made me think that Thorium could be the browser for older operating systems. If someone did provide the patches to port Thorium to Mac OS 10.5, would you accept them?

**Comment by gz83** [comment]
There are currently no plans to do so.
